### PR TITLE
Module 1: Week 2(D4) - Portfolio Project

### PR DESCRIPTION
**Pull request by Strangeal**
### Desktop View - Portfolio Project

Added media queries

### CODE REVIEW

File: header.html [hunk 1]
```diff
@@ -13,26 +13,26 @@
...
         <!--Front Page-->
         <section class="dev">
-            <h1>I’m Mariane
-                Glad to see you!
-            </h1>
-                <p>
-                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-                </p>
+            <div class="dev-child">
+                <h1>I’m Justice Glad to see you!</h1>
```
Comment (Nemwel-Boniface) [inline header.html:34]: - [x] Kindly note that you are required to stick as much as possible to the provided Figma design in fonts, colors, etc. The font size used is 6rem which makes it around 96px. Kindly change it to 48px as specified in the Figma design

File: header.html [hunk 2]
```diff
@@ -13,26 +13,26 @@
...
-        <img src="img/header-shapesmobile@2x.svg" alt="" class="bg-shapes">
-            <nav>
-                <ul class="navigation">
-                    <li class="logo">
-                        <a href="#">My Logo</a>
-                    </li>
+            <nav class="navigation">
+                <p class="logo">
+                    <a href="#">My Logo</a>
+                </p>
```
Comment (Nemwel-Boniface) [inline header.html:19]: - [x] Kindly note that you are required to stick as much as possible to the provided Figma design in fonts, colors, etc. The font size for this logo should be 18px as specified in the Figma design.

File: header.html [hunk 3]
```diff
@@ -13,26 +13,26 @@
...
         <section class="dev">
-            <h1>I’m Mariane
-                Glad to see you!
-            </h1>
-                <p>
-                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-                </p>
+            <div class="dev-child">
+                <h1>I’m Justice Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```
Comment (Nemwel-Boniface) [inline header.html:35]: - [x] Kindly note that you are required to stick as much as possible to the provided Figma design in fonts, colors, etc. The font size for this p should be 20px as specified in the Figma design.

File: header.html [hunk 4]
```diff
@@ -13,26 +13,26 @@
...
                 <a href="#">LET’S CONNECT</a>
                 <ul class="social-icons">
```
Comment (Nemwel-Boniface) [inline header.html:37]: - [x] Kindly note that you are required to stick as much as possible to the provided Figma design in fonts, colors, etc. Kindly reduce the size of these icons to match the provided Figma design.

File: header.html [hunk 5]
```diff
@@ -41,96 +41,99 @@ <h1>I’m Mariane
...
+        <section class="card card-reverse">
+            <img src="img/Snapshoot-Portfolio1.svg" alt="Snapshoot-Portfolio" class="float-img">
+            <div class="card-block">
+                <h2>Multi-Post Stories</h2>
+                <ul class="card-frame">
+                    <li class="no-count">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```
Comment (Nemwel-Boniface) [inline header.html:75]: - [x] Kindly note that you are required to stick as much as possible to the provided Figma design in fonts, colors, etc. The font size for this p should be 16px as specified in the Figma design.

**Kindly implement this on all the ```<p>``` for the works section**

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Strangeal           :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done an amazing job with your portfolio desktop version so far :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [ ] Kindly ensure that you stick as much as possible to the chosen Figma design in font sizes, font colors etc.

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @Strangeal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [x] No linters errors ✔️ 
- [x] Good implementation of required changes ✔️ 
- [x] Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: header.html [hunk 6]
```diff
@@ -145,19 +148,20 @@ <h2>About Myself</h2>
...
                 </ul>
                 </div>
-            </section>
-            <!-- Contact form -->
-            <section class="form-group">
-                <img src="img/contact-form-shapes-mobile.svg" alt="Background shapes design" class="form-bg-img">
+        </section>
+        <!-- Contact form -->
+        <section class="form-group">
+            <div class="form-child">
```
Comment (Nedramrez) [inline header.html:154]: Pleadse check the padding of `"form-child" <div>` and try to make it closer to the Figma design.

File: header.html (hunk 1 above)
Comment (Nedramrez) [inline header.html:34]: Please check the` padding-top` property of the `"dev-child" <div>` so your headline section would be as it is at Figma design.

**Review by Nedramrez — COMMENTED** [review]
Nice job!

**Pull request merged**